locales/de.lua

```diff
@@ -0,0 +1,134 @@
+        bleed_out_help = 'Du blutest aus in: ~r~%{time}~s~ Sekunden, dir kann geholfen werden',
+        request_help = 'Drücke [~r~G~s~] für eine hilfeanfrage',
+        help_requested = 'Rettungsdienst wurde verständigt',
+        amb_plate = 'AMBU', -- Should only be 4 characters long due to the last 4 being a random 4 digits
+        heli_plate = 'LIFE',  -- Should only be 4 characters long due to the last 4 being a random 4 digits
+        status = 'Status Prüfung',
+        is_staus = 'Ist %{status}',
+        healthy = 'Du bist wieder ganz gesund!',
+        safe = 'Krankenhaus Safe',
+        pb_hospital = 'Pillbox Krankenhaus',
+        pain_message = 'Dein %{limb} fühlt sich %{severity} an',
+        many_places = 'Du hast schmerzen an verschiedenen Stellen...',
+        bleed_alert = 'Du bist %{bleedstate}',
+        ems_alert = 'Rettungsdienst Alarm - %{text}',
+        mr = 'Herr.',
+        mrs = 'Frau.',
+        dr_needed = 'Ein Doktor wird im Pillbox Krankenhaus benötigt',
+        ems_report = 'Rettungsdienst Report',
+        message_sent = 'Nachricht zu senden',
+        check_health = 'Prüfe die Gesunheit eines Spielers',
+        heal_player = 'Heile einen Spieler',
+        revive_player = 'Belebe einen Spieler',
+        revive_player_a = 'Belebe einen Spieler oder dich selbst (Admin Only)',
+        player_id = 'Spieler ID (may be empty)',
+        pain_level = 'Setzt dein oder eines Spielers schmerzlevel (Admin Only)',
+        kill = 'Töte einen Spieler oder dich selbst (Admin Only)',
+        heal_player_a = 'Keile einen Spieler oder dich selbst (Admin Only)',
+    },
+    mail = {
+        sender = 'Pillbox Krankenhaus',
+        subject = 'Krankenhauskosten',
+        message = 'Hallo %{gender} %{lastname}, <br /><br />Hiermit erhalten Sie eine Email über die Kosten ihres letzten Krankenhaus besuchs.<br />Die gesammt Kosten betragen: <strong>$%{costs}</strong><br /><br />Wir wünschen eine schnelle Genesung!'
+    },
+    states = {
+        irritated = 'irritiert',
+        quite_painful = 'leicht schmerzhaft',
+        painful = 'schmerzhaft',
+        really_painful = 'sehr schmerzhaft',
+        little_bleed = 'wenig blutend...',
+        bleed = 'blutend...',
+        lot_bleed = 'etwas mehr blutend...',
+        big_bleed = 'sehr stark blutend...',
+    },
+    menu = {
+        amb_vehicles = 'Rettungsdienst Fahrzeuge',
+        close = '⬅ Schließe Menu',
+    },
+    text = {
+        pstash_button = '~g~E~w~ - Persönliches Fach',
+        pstash = 'Persönliches Fach',
+        onduty_button = '~g~E~w~ - Dienst antretten',
+        offduty_button = '~r~E~w~ - Dienst quittieren',
+        duty = 'In/Ausser Dienst',
+        armory_button = '~g~E~w~ - Waffenschrank',
+        armory = 'Waffenschrank',
+        storeveh_button = '~g~E~w~ - Verwahre Fahrzeug',
+        veh_button = '~g~E~w~ - Fahrzeuge',
+        storeheli_button = '~g~E~w~ - Verwahre Helikopter',
```

Comment:
```suggestion
        storeheli_button = '~g~E~w~ - Helikopter einparken',
```

Even if its sounds weird, thats the official term